src/pages/contract/ContractPage.tsx

```diff
@@ -88,6 +92,8 @@ export function ContractPage(): JSX.Element {
         setInstantiationTx(first);
       })
       .catch(() => setInstantiationTx(errorState));
+
+    clientContext.client.getContractCodeHistory(contractAddress).then(setContractCodeHistory);
```

Comment:
Here we should at least catch and log errors with:

```ts
      .catch((error) => {
        console.error(error);
      });
```

No need for a UI now but then we can easily add one